pkg/controllers/namespaceregistration/reconcile_test.go

```diff
@@ -56,7 +56,7 @@ var _ = Describe("Reconcile", func() {
 		state, err = testenv.InitResources(ctx, "./testdata/reconcile/test1")
 		Expect(err).ToNot(HaveOccurred())
 
-		namespaceRegistration := state.GetNamespaceRegistration("test-namespace-1")
+		namespaceRegistration := state.GetNamespaceRegistration("cu-test-namespace-1")
```

Comment:
Should't we use our constant for `cu-` here?

Reply:
done